app/views/components/_contents-list-with-body.html.erb

```diff
@@ -0,0 +1,15 @@
+<% contents ||= [] %>
+<div id="contents" data-module="sticky-element-container">
```

Comment:
Do we ever need to render this component with no `contents`? If not,  could some of the `if contents.any?` checks below be moved to wrap the whole component instead?

Reply:
It's a good point and something I am wondering about myself...

At the moment with no contents and a block the block is yielded (rendered) which I think is necessary for content items which have no `details[:contents]`, these def exist, I'll try and find a specialist document by way of example.

With contents and no block the content list is rendered - this probably isn't what we want as it's a content list with no content.

Reply:
@vanitabarrett I need to add component tests for the above scenarios.